language-server/modules/langserver-core/src/test/resources/completion/statement_context/config/onfail_clause_ctx_config2c.json

```diff
@@ -254,6 +254,102 @@
       "filterText": "var",
       "insertText": "var ",
       "insertTextFormat": "Snippet"
+    },
```

Comment:
@gimantha, why do we need this change in this PR? It just reverts the previous lang PR (https://github.com/ballerina-platform/ballerina-lang/pull/34817)? Was it already failing on 2201.0.x?

Reply:
Yes. it failed. https://github.com/ballerina-platform/ballerina-lang/runs/5890888206?check_suite_focus=true

Reply:
Thanks. 

Seems to have been failing in https://github.com/ballerina-platform/ballerina-lang/runs/5880386248?check_suite_focus=true also. cc @mohanvive @malinthar